packages/salesforce/src/Adaptor.js

```diff
@@ -303,26 +300,19 @@ const defaultOptions = {
  * @function
  * @param {String} qs - A query string.
  * @param {Object} options - Options passed to the bulk api.
- * @param {integer} [options.pollTimeout] - Polling timeout in milliseconds.
- * @param {integer} [options.pollInterval] - Polling interval in milliseconds.
+ * @param {integer} [options.pollTimeout=90000] - Polling timeout in milliseconds.
+ * @param {integer} [options.pollInterva=3000] - Polling interval in milliseconds.
  * @param {Function} callback - A callback to execute once the record is retrieved
+ * @throws {Error} Throws an error if the execution exceeds the specified `pollTimeout` duration.
```

Comment:
Again this is fine but I bet there are more things to make this throw, so it's misleading

Reply:
Okay i am going to remove all `@throws` for now